x-pack/plugins/maps/public/classes/sources/es_search_source/util/scaling_documenation_popover.tsx

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { Component, Fragment } from 'react';
+import { EuiButtonIcon, EuiPopover, EuiPopoverTitle, EuiText } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+
+interface Props {
+  limitOptionLabel: string;
+  clustersOptionLabel: string;
+  maxResultWindow: string;
+  mvtOptionLabel: string;
+}
+
+interface State {
+  isPopoverOpen: boolean;
+}
+
+export class ScalingDocumenationPopover extends Component<Props, State> {
+  state: State = {
+    isPopoverOpen: false,
+  };
+
+  _togglePopover = () => {
+    this.setState((prevState) => ({
+      isPopoverOpen: !prevState.isPopoverOpen,
+    }));
+  };
+
+  _closePopover = () => {
+    this.setState({
+      isPopoverOpen: false,
+    });
+  };
+
+  _renderContent() {
+    return (
+      <div>
+        <EuiPopoverTitle>
+          <FormattedMessage id="xpack.maps.scalingDocs.title" defaultMessage="Scaling" />
+        </EuiPopoverTitle>
+        <EuiText grow={false}>
+          <p>
+            <FormattedMessage
+              id="xpack.maps.scalingDocs.intro"
+              defaultMessage="Select the appropriate option for your use case."
+            />
+          </p>
+          <dl>
+            <dt>{this.props.limitOptionLabel}</dt>
+            <dd>
+              <p>
+                <FormattedMessage
+                  id="xpack.maps.scalingDocs.limitDetails"
+                  defaultMessage="Layer displays features from the first {maxResultWindow} documents. Results exceeding {maxResultWindow} are not displayed."
```

Comment:
Does this say the same thing? So you don't need the second sentence.

The layer displays features from the first {maxResultWindow} documents only.

Or if you don't need to use the term "the layer"

Displays features from the first {maxResultWindow} documents only.